packages/rich-text/src/plugins/Hyperlink/index.tsx

```diff
@@ -72,12 +72,13 @@ function UrlHyperlink(props: HyperlinkElementProps) {
   const isReadOnly = useReadOnly();
   const sdk: FieldExtensionSDK = useSdkContext();
   const { uri } = props.element.data;
+  const tracking = useTrackingContext();
```

Comment:
Do we still need the `TrackingProvider` component and the `useTrackingContext` when we have things available on `editor.tracking`?

Reply:
Yeah, I think so.

Reply:
Cool. We can make the tracking plugin accept `onAction` and refactor the code accordingly

Reply:
Yep.